frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -49,5 +49,19 @@ common_info.report_bug = {
         })
     end
 }
+if Device:isKobo() then
+    common_info.reboot = {
+        text = _("Reboot the device"),
+        callback = function()
+            UIManager:scheduleIn(0, UIManager.reboot_action)
+        end
+    }
+    common_info.poweroff = {
+        text = _("Power off"),
+        callback = function()
+            UIManager:scheduleIn(0, UIManager.poweroff_action)
```

Comment:
Last nitpick, can you replace `UIManager:scheduleIn(0, foo)` with `UIManager:nextTick(foo)`?